assets/app/view/game/tile.rb

```diff
@@ -67,7 +67,8 @@ def render
         children << render_tile_part(Part::Blocker)
         rendered_loc_name = render_tile_part(Part::LocationName) if @tile.location_name && (@tile.cities.size <= 1)
         @tile.reservations.each { |x| children << render_tile_part(Part::Reservation, reservation: x) }
-        children << render_tile_part(Part::Icons) unless @tile.icons.empty?
+        children << render_tile_part(Part::Icons) unless @tile.icons.reject(&:large).empty?
```

Comment:
large, small = tile.icons.partition(&:large)

Reply:
Yeah, I had that at one point.